<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiPass</class>
 <widget class="QMainWindow" name="MultiPass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>996</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiPass</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(34, 40, 49);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>904</width>
      <height>621</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vertical">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizont_btns">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizont_add_space">
         <item>
          <widget class="QPushButton" name="btn_add">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Britannic Bold</family>
             <pointsize>30</pointsize>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(68, 73, 80);
background-color: rgb(238, 238, 238);</string>
           </property>
           <property name="text">
            <string>add</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="space_add">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>118</width>
             <height>60</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_new">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Britannic Bold</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(68, 73, 80);
background-color: rgb(238, 238, 238);</string>
         </property>
         <property name="text">
          <string>new account</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="space_new_delete">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_delete">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Britannic Bold</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(68, 73, 80);
background-color: rgb(238, 238, 238);</string>
         </property>
         <property name="text">
          <string>delete account</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frame_names">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(57, 62, 70);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>901</width>
          <height>34</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_main" stretch="0,2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_tab">
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
           <item>
            <widget class="QLabel" name="label_tab">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Britannic Bold</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(189, 190, 192);</string>
             </property>
             <property name="text">
              <string>  site</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_search">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(189, 190, 192);
background-color: rgb(238, 238, 238);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_search">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>s</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_values">
           <item>
            <spacer name="space_values">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_email">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Britannic Bold</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(189, 190, 192);</string>
             </property>
             <property name="text">
              <string>email</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="space_e_p">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>88</width>
               <height>27</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_password">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Britannic Bold</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(189, 190, 192);</string>
             </property>
             <property name="text">
              <string>password</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="space_e_p_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>88</width>
               <height>27</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizont_data">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea_tab">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>510</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>260</width>
           <height>520</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(57, 62, 70);
border: 1px solid rgb(57,62,70);</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_tab">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>248</width>
            <height>518</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_values">
         <property name="minimumSize">
          <size>
           <width>650</width>
           <height>510</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>650</width>
           <height>520</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(238, 238, 238);</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_values">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>648</width>
            <height>518</height>
           </rect>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>651</width>
             <height>34</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="value_copy_1">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="value_Class_1">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>11111</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(238, 238, 238);
font: 25pt &quot;Britannic Bold&quot;;
color: rgb(68, 73, 80);
border:none;</string>
                </property>
                <property name="text">
                 <string>asdasdasd</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_Class_1">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(238, 238, 238);
color: rgb(34, 40, 49);
font: 25pt &quot;Britannic Bold&quot;;
border:none;</string>
                </property>
                <property name="text">
                 <string>k</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="value_copy_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="value_Class_2">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>11111</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(238, 238, 238);
font: 25pt &quot;Britannic Bold&quot;;
color: rgb(68, 73, 80);
border:none;</string>
                </property>
                <property name="text">
                 <string>asdasdasd</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_Class_2">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(238, 238, 238);
color: rgb(34, 40, 49);
font: 25pt &quot;Britannic Bold&quot;;
border:none;</string>
                </property>
                <property name="text">
                 <string>k</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>340</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>279</width>
       <height>139</height>
      </rect>
     </property>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>249</width>
        <height>42</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_ico_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(238, 238, 238);
font: 25pt &quot;Britannic Bold&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_tab_name_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(238, 238, 238);
color: rgb(34, 40, 49);
font: 25pt &quot;Britannic Bold&quot;;</string>
         </property>
         <property name="text">
          <string>Youtube</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>640</y>
      <width>411</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="value_copy">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="value_Class">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>11111</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Britannic Bold</family>
         <pointsize>25</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 238);
font: 25pt &quot;Britannic Bold&quot;;
color: rgb(68, 73, 80);</string>
       </property>
       <property name="text">
        <string>asdasdasd</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Class">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>77-Essential-Icons</family>
         <pointsize>20</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 238);
color: rgb(34, 40, 49);
border:none;</string>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
